assets/stylesheets/bootstrap/_alerts.scss

```
@@ -11,9 +11,27 @@
   margin-bottom: $line-height-computed;
   border: 1px solid transparent;
   border-radius: $alert-border-radius;
+  display: flex;
+
+  &.alert-vertical {
+    flex-direction: column;
+
+    > button {
+      margin-top: 10px;
```

ちょっと細かいですが12pxにするとテキスト周りのスペースが統一されてよりカッチリ見えそうです

```suggestion
      margin-top: 12px;
```